tests/integration/test_reload_max_table_size_to_drop/test.py

```diff
@@ -22,7 +26,21 @@ def start_cluster():
         cluster.shutdown()
 
 
-def test_reload_max_table_size_to_drop(start_cluster):
+@pytest.fixture
+def reset_config():
```

Comment:
AFAICS it is required because of substitution in `test_reload_max_table_size_to_drop`, maybe it is better to adjust this test instead?